src/background/api.ts

```diff
@@ -55,8 +55,15 @@ export const fetchTwitterHandle = async (tabId: number, host: string, dispatchBa
     dispatchBackgroundActions.fetchHandleStart({ tabId })
     const res = await makeHandleRequest(host)
     const { twitter_handle } = await res.json()
+
+    //testing fetch response status
+    // const { status, statusText, url } = res
+    // console.log('fetchTwitterHandel request Status:', status, statusText, url)
+
     return dispatchBackgroundActions.fetchHandleSuccess({ tabId, host, handle: twitter_handle })
   } catch (error) {
-    return dispatchBackgroundActions.fetchHandleFailure({ tabId, host, error })
+    let alert = error.name
+
+    return dispatchBackgroundActions.fetchHandleFailure({ tabId, host, alert })
```

Comment:
See notes below for `logErrorAndReturnAlert`. If those are implemented, then we could instead write here

```typescript
return dispatchBackgroundActions.fetchHandleFailure({ tabId, host, alert: logErrorAndReturnAlert(error) })
```

Reply:
Nice!